cmd/mode.go

```diff
@@ -29,32 +29,39 @@ type Mode struct {
 	}
 }
 
-//
 // Build assets.
 func (r *Mode) Build(application *api.Application) (err error) {
+	binDir := path.Join(BinDir, "maven")
+	maven := &repository.Maven{
+		M2Dir:  M2Dir,
+		BinDir: binDir,
+		Remote: repository.Remote{
+			Identities: application.Identities,
+		},
+	}
+	path, err := maven.writeSettingsFile()
```

Comment:
Shouldn't this be maven.CreateSettingsFile()
I don't think this method exists and isn't exported.  How does this compile?

Reply:
This should be updated; this should never have worked my bad